### ISSUE TITLE
Example bug

### ISSUE THREAD

**Issue by sgress454** [sgress454/github-api-tester-uno#1]
comment

**Comment by sgress454** [comment]
Thanks for posting, @sgress454.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/sgress454/github-api-tester-uno/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sgress454** [comment]
Thanks for posting, @sgress454.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/sgress454/github-api-tester-uno/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sgress454** [comment]
: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/sgress454/github-api-tester-uno/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by sgress454** [comment]
: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/sgress454/github-api-tester-uno/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by sgress454** [comment]
: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/sgress454/github-api-tester-uno/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by sgress454** [comment]
: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive, with any new information you can provide.

If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by sgress454** [comment]
: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).

If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by sgress454** [comment]
@sgress454,sgress454: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).

If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by sgress454** [comment]
@sgress454: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).

If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by sailsbot** [comment]
commentz iz fun

**Comment by sgress454** [comment]
@sgress454,@sailsbot: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by sgress454** [comment]
@sgress454,@sailsbot: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).

If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by sgress454** [comment]
Thanks for posting, @sgress454.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please:
- review our [contribution guide](https://github.com/sgress454/github-api-tester-uno/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
- create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sgress454** [comment]
eine comment

**Comment by sgress454** [comment]
eine oder comment

**Comment by sgress454** [comment]
should remove label...

**Comment by sgress454** [comment]
helloo

**Comment by sgress454** [comment]
foo

**Comment by sgress454** [comment]
barz

**Comment by sgress454** [comment]
make lambda go now?

**Comment by sgress454** [comment]
how about now?

**Comment by sgress454** [comment]
now?

**Comment by sgress454** [comment]
WOOOOT!

**Comment by sgress454** [comment]
gimme some sailsbot!!!

**Comment by sgress454** [comment]
um what?

**Comment by sgress454** [comment]
oh sorry sailsbot here ya go

**Comment by sgress454** [comment]
BOOM
